pennylane/labs/resource_estimation/ops/op_math/controlled_ops.py

```diff
@@ -379,11 +798,12 @@ def _resource_decomp(**kwargs) -> Dict[re.CompressedResourceOp, int]:
 
     @staticmethod
     def textbook_resource_decomp() -> Dict[re.CompressedResourceOp, int]:
-        r"""Resources for the Toffoli gate
+        r"""Returns a dictionary representing the resources of the operator. The
+        keys are the operators and the associated values are the counts.
 
         Resources:
-            The resources are taken (figure 4.9) from the textbook `Quantum Computation and Quantum Information
-            <https://www.cambridge.org/highereducation/books/quantum-computation-and-quantum-information/01E10196D0A682A6AEFFEA52D53BE9AE#overview>`_.
+            The resources are taken (figure 4.9) from the textbook `Quantum Computation and
```

Comment:
```suggestion
            The resources are taken (Figure 4.9) from the textbook `Quantum Computation and
```

Also we should include the authors' names. Probably more than 1 book out there with the name Quantum Computation and Quantum Information.